src/daemon/dlt.conf

```diff
@@ -81,6 +81,56 @@ RingbufferStepSize = 500000
 # Read gateway configuration from another location
 # GatewayConfigFile = /etc/dlt_gateway.conf
 
+
+########################################################################
```

Comment:
These explanations are too long for dlt.conf and we don't need hypothesis (or just summarize it in commit message). Please have a short description for this and add it to dlt.conf.md under doc instead.

Reply:
done